setup.cfg

```diff
@@ -0,0 +1,66 @@
+[mypy]
+pretty = True
+allow_redefinition = True
+check_untyped_defs = True
+disallow_any_generics = True
+disallow_untyped_calls = True
+exclude = venv|migrations|tests|test.py|test_\w+\.py
+ignore_errors = False
+ignore_missing_imports = True
+implicit_reexport = True
+local_partial_types = True
+no_implicit_optional = True
+plugins = sqlalchemy.ext.mypy.plugin
```

Comment:
Isn't it `sqlalchemy-stubs` which is abandoned and no longer recommended?

Reply:
It's not stubs, it's `sqlalchemy[mypy]`.

Reply:
Btw I don't know how to define features in Pipfile, so it's not listed in Pipfile.